src/FondOfSpryker/Yves/GoogleTagManagerStoreConnector/Plugin/DataLayer/StoreDataLayerExpanderPlugin.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerStoreConnector\Plugin\DataLayer;
+
+use FondOfSpryker\Yves\GoogleTagManagerExtension\Dependency\GoogleTagManagerDataLayerExpanderPluginInterface;
+use Spryker\Yves\Kernel\AbstractPlugin;
+
+/**
+ * @method \FondOfSpryker\Yves\GoogleTagManagerStoreConnector\GoogleTagManagerStoreConnectorFactory getFactory()
+ */
+class StoreDataLayerExpanderPlugin extends AbstractPlugin implements GoogleTagManagerDataLayerExpanderPluginInterface
+{
+    /**
+     * @param string $pageType
+     * @param array $twigVariableBag
+     *
+     * @return bool
+     */
+    public function isApplicable(string $pageType, array $twigVariableBag = []): bool
+    {
+        return true;
+    }
+
+    /**
+     * @param string $page
+     * @param array $twigVariableBag
+     * @param array $variableList
+     *
+     * @return array
+     */
+    public function expand(string $page, array $twigVariableBag, array $variableList): array
```

Comment:
Use $dataLayer instead of $variableList. It is a **DataLayer**ExpanderPluginInterface.